packages/api-demo/package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "scripts": {
     "serve:no-browser": "npm run serve -- --no-browser",
-    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/containerjs-bundle.js --mount=/resources/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
+    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api:node_modules/containerjs-api-browser/build/dist/containerjs-api.js --mount=/resources/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
```

Comment:
That option was to make copying/creating a `notification.html` file easier, as it copies our notification into a directory. Unfortunately, this notification still needs to be served as it did before.